run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+cd /home/ec2-user/nina-indexer
+pm2 stop nina-indexer && pm2 delete nina-indexer
+pm2 start yarn --interpreter bash --name nina-indexer -- start:indexer
```

Comment:
we will also want to start api service here

Reply:
@facing-n yep, added.